app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -155,122 +155,136 @@ class AppVersionActivityTest {
     )
   }
 
-  @After
-  fun tearDown() {
-    testCoroutineDispatchers.unregisterIdlingResource()
-    Intents.release()
-  }
-
-  private fun setUpTestApplicationComponent() {
-    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+  @Test
+  fun testAppVersionActivity_hasCorrectActivityLabel() {
+    ActivityScenario.launch(AppVersionActivity::class.java).use { scenario ->
+      scenario.onActivity { activity ->
+        val title = activity.title
+        assertThat(title).isEqualTo(context.getString(R.string.app_version_activity_title))
+      }
+    }
   }
 
   @Test
   fun testActivity_createIntent_verifyScreenNameInIntent() {
     val screenName = createAppVersionActivityIntent().extractCurrentAppScreenName()
-
     assertThat(screenName).isEqualTo(ScreenName.APP_VERSION_ACTIVITY)
   }
 
   @Test
   fun testAppVersionActivity_loadFragment_displaysAppVersion() {
-    launchAppVersionActivityIntent().use { scenario ->
-      val lastUpdateDate = scenario.convertTimeStampToDate(context.getLastUpdateTime())
-      onView(
-        withText(
-          String.format(
-            context.resources.getString(R.string.app_version_name),
-            context.getVersionName()
-          )
-        )
-      ).check(matches(isDisplayed()))
-      onView(
-        withText(
-          String.format(
-            context.resources.getString(R.string.app_last_update_date),
-            lastUpdateDate
-          )
-        )
-      ).check(
-        matches(isDisplayed())
-      )
-    }
-  }
-
-  @Test
-  fun testAppVersionActivity_configurationChange_appVersionIsDisplayedCorrectly() {
-    launchAppVersionActivityIntent().use { scenario ->
-      onView(isRoot()).perform(orientationLandscape())
-      val lastUpdateDate = scenario.convertTimeStampToDate(context.getLastUpdateTime())
-      onView(
-        withId(
-          R.id.app_version_text_view
-        )
-      ).check(
-        matches(
+    ActivityScenario.launch(AppVersionActivity::class.java).use { scenario ->
+      scenario.onActivity { activity ->
```

Comment:
This lambda appears unused, please move the assertions up inside the `use` lambda and remove `onActivity`

Reply:
Ditto below